### DISCUSSION TITLE
Is it possible to remove duplicates from "which" suggestions?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [sxyazi/yazi#1332] [1. Q&A] [answered]
I replaced some standard mappings with my own, but "which" shows both of them, even though the second one won't be ever used. Is it possible to remove those duplicates?

**Comment by sxyazi** [comment, marked as answer]
It was a bug and fixed in https://github.com/sxyazi/yazi/pull/975, the latest main branch is needed atm

**Reply by wojciech-kulik** [reply]
I confirm, it's fixed on the main branch 🔥
